tests/backup/backup_namespace_labelled_test.go

```diff
@@ -199,3 +201,225 @@ var _ = Describe("{NamespaceLabelledBackupSharedWithDifferentAccessMode}", func(
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudCredName, cloudCredUID, ctx)
 	})
 })
+
+// BackupScheduleForOldAndNewNS Schedule backup using namespace label for old namespace and new namespace
+var _ = Describe("{BackupScheduleForOldAndNewNS}", func() {
+	var (
+		numberOfNewNamespaces  int
+		scheduleInterval       int64
+		err                    error
+		backupLocationUID      string
+		cloudCredUID           string
+		clusterUid             string
+		credName               string
+		backupLocationName     string
+		restoreName            string
+		periodicSchPolicyName  string
+		periodicSchPolicyUid   string
+		scheduleName           string
+		nsLabelString          string
+		schBackupAfterAddingNS string
+		newNamespaces          []string
+		bkpNamespaces          []string
+		cloudCredUidList       []string
+		allScheduleBackupNames []string
+		restoreNames           []string
+		nsLabelsMap            map[string]string
+		contexts               []*scheduler.Context
+		appContexts            []*scheduler.Context
+	)
+	labelSelectors := make(map[string]string)
+	backupLocationMap := make(map[string]string)
+	namespaceMapping := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	JustBeforeEach(func() {
+		StartTorpedoTest("BackupScheduleForOldAndNewNS", "Schedule backup with old and new namespace using namespace label", nil, 84852)
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < 3; i++ {
```

Comment:
Declare a variable `initialNamespaces` and assign `3` to it so that we know why we need 3 and also it would be easier to increase and decrease in future.

Reply:
Done